<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCircuit</class>
 <widget class="QMainWindow" name="frmCircuit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>405</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>物理场-电路</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>边界条件列表</string>
    </property>
    <widget class="QTableWidget" name="tbFaces">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>381</width>
       <height>291</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>面编号</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>源/负载</string>
      </property>
      <property name="textAlignment">
       <set>AlignJustify|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>边界条件</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>315</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemove">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>315</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>移除选中行</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
